examples/demos/jobs.yml

```diff
@@ -48,14 +62,24 @@
           cohesity_validate_certs: "{{ var_validate_certs }}"
           cohesity_protection:
               state: absent
-              job_name: "{{ hostvars[item]['ansible_host'] }}"
-              sources:
-                - endpoint: "{{ hostvars[item]['ansible_host'] }}"
+              job_name: "protect_physical_linux"
+              delete_backups: True
+
+      - name: Remove Protection Job for Physical windows server
+        include_role:
+          name: cohesity.cohesity_ansible_role
+          tasks_from: job
+        vars:
+          cohesity_server: "{{ var_cohesity_server }}"
+          cohesity_admin: "{{ var_cohesity_admin }}"
+          cohesity_password: "{{ var_cohesity_password }}"
+          cohesity_validate_certs: "{{ var_validate_certs }}"
+          cohesity_protection:
+              state: absent
+              job_name: "protect_physical_windows"
               delete_backups: True
-        with_items: "{{ groups.physical }}"
-        tags: [ 'cohesity', 'jobs', 'remove', 'physical' ]
 
-      - name: Create new Protection Jobs for each Physical Server
+      - name: Create new Protection Job for Physical linux Servers
```

Comment:
Linux Physical server or Physical linux server? It can be consistent in all places